test/drafts/Strings.test.js

```diff
@@ -0,0 +1,49 @@
+const { constants } = require('openzeppelin-test-helpers');
+
+const StringsMock = artifacts.require('StringsMock');
+
+contract('Strings', function () {
+  const a = 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+  const b = 'BBBBBBBBBBBBBBBBBBBBBBB';
+  const ab = a + b;
+  const aa = a + a;
+  const empty = '';
+  const zero = 0;
+  const positiveNumber = 3195;
+
+  beforeEach(async function () {
+    this.strings = await StringsMock.new();
+  });
+
+  describe('concatenate', function () {
+    it('concatenates strings', async function () {
+      (await this.strings.concatenate(a, b)).should.equal(ab);
+    });
+
+    it('concatenates a string with itself', async function () {
```

Comment:
For this test, we should add a function to the mock contract that receives a single string and uses it for both arguments of `Strings.concatenate`, as opposed to passing two strings that happen to be equal.